sample-docker-app/package.json

```diff
@@ -11,7 +11,9 @@
     "build": "tsc",
     "start": "ts-node index",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build:image": "npm run build && docker build -t sample-docker-app ."
+    "build:images": "npm run build && npm run build:image:x86_64 && npm run build:image:arm64v8",
+    "build:image:x86_64": "docker build -t sample-docker-app-x86_64 .",
+    "build:image:arm64v8": "docker build -f Dockerfile.arm64v8 -t sample-docker-app-arm64v8 ."
```

Comment:
I took the choice to rename the existing commands to build, not sure you will agree with that.

Reply:
Probably I'm wrong, but will this work on a system that can't run ARM64 containers?

up to now I use buildx to build it for linux/arm64 
`docker buildx build --platform linux/arm64 -t sample-docker-app-arm64v8  -f Dockerfile.arm64v8 --load .`

Reply:
I just came across `docker buildx` when reading your comments, it's clearly a worthy alternative.
The build for ARM64 is working on MacOS with Docker Desktop at least ;)